apps/examples/rmc/rmc_main.c

```diff
@@ -0,0 +1,181 @@
+#include <ble_manager/ble_manager.h>
+
+static void ble_scan_state_changed_cb(ble_client_scan_state scan_state)
+{
+	printf("[DAE] '%s' is called\n", __FUNCTION__);
+	return;
+}
+
+static void ble_device_scanned_cb(ble_client_scanned_device *scanned_device)
+{
+	printf("[DAE] '%s' is called\n", __FUNCTION__);
+	return;
+}
+
+static void ble_device_disconnected_cb(ble_conn_handle conn_id)
+{
+	printf("[DAE] '%s' is called\n", __FUNCTION__);
+	return;
+}
+
+static void ble_device_connected_cb(ble_client_device_connected *connected_device)
+{
+	printf("[DAE] '%s' is called\n", __FUNCTION__);
+	return;
+}
+
+static void ble_operation_notification_cb(ble_client_operation_handle *handle, ble_data *read_result)
+{
+	printf("[DAE] '%s' is called\n", __FUNCTION__);
+	return;
+}
+
+static void ble_server_connected_cb(ble_conn_handle con_handle, ble_server_connection_type_e conn_type, uint8_t mac[BLE_BD_ADDR_MAX_LEN])
+{
+	printf("[DAE] '%s' is called\n", __FUNCTION__);
+	return;
+}
+
+static void utc_cb_charact_a_1(ble_server_attr_cb_type_e type, ble_conn_handle linkindex, ble_attr_handle handle, void *arg)
+{
+	char *arg_str = (char *)arg;
+	printf("-- [DAE CHAR] %s [ type : %d / handle : %d / attr : %d ] --\n", arg_str, type, linkindex, handle);
+}
+
+static void utc_cb_desc_b_1(ble_server_attr_cb_type_e type, ble_conn_handle linkindex, ble_attr_handle handle, void *arg)
+{
+	char *arg_str = (char *)arg;
+	printf("-- [DAE DESC] %s [ type : %d / handle : %d / attr : %d ] --\n", arg_str, type, linkindex, handle);
+}
+
+static ble_server_gatt_t gatt_profile[] = {
+	{
+		.type = BLE_SERVER_GATT_SERVICE,
+		.uuid = {0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x01, 0x01, 0x01, 0x01},
+		.uuid_length = 16,
+		.attr_handle = 0x006a,
+	},
+
+	{.type = BLE_SERVER_GATT_CHARACT, .uuid = {0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0x01, 0x01, 0x01, 0x02}, .uuid_length = 16, .property = BLE_ATTR_PROP_RWN | BLE_ATTR_PROP_WRITE_NO_RSP, .permission = BLE_ATTR_PERM_R_PERMIT | BLE_ATTR_PERM_W_PERMIT, .attr_handle = 0x006b, .cb = utc_cb_charact_a_1, .arg = NULL},
+
+	{.type = BLE_SERVER_GATT_DESC, .uuid = {0x02, 0x29}, .uuid_length = 2, .permission = BLE_ATTR_PERM_R_PERMIT | BLE_ATTR_PERM_W_PERMIT, .attr_handle = 0x006c, .cb = utc_cb_desc_b_1, .arg = NULL},
+};
+
+/****************************************************************************
+ * rmc_main
+ ****************************************************************************/
+int rmc_main(int argc, char *argv[])
+{
+	printf("[RMC] test!!\n");
+
+	ble_result ret;
+
+	ble_client_init_config client_config = {
+		ble_scan_state_changed_cb,
+		ble_device_scanned_cb,
+		ble_device_disconnected_cb,
+		ble_device_connected_cb,
+		ble_operation_notification_cb,
+		240};
+
+	ble_server_init_config server_config = {
+		ble_server_connected_cb,
+		true,
+		gatt_profile, sizeof(gatt_profile) / sizeof(ble_server_gatt_t)};
+
+	if (argc < 2) {
+		return 0;
```

Comment:
How about printing the usage?
If `argc` is smaller than 2, so it returns, user cannot know it runs successfully or not.

Reply:
Ok, after I implement every functions of BLE tc, I will add usage.